proto/c1/connector/v2/resource.proto

```
@@ -74,6 +75,20 @@
   rpc ListResources(ResourcesServiceListResourcesRequest) returns (ResourcesServiceListResourcesResponse);
 }
 
+// AccountCreationStatusService is used in conjunction with create account flows to confirm confirmation or status of created accounts
+// For example, github would send an invite as part of its flow and return "poll for invite acceptance" with a task id
+service AccountCreationStatusService {
+  rpc GetAccountCreationStatus(GetAccountCreationStatusRequest) returns (AccountCreationStatusResponse);
```

Buf (1.47.2)

RPC response type "AccountCreationStatusResponse" should be named "GetAccountCreationStatusResponse" or "AccountCreationStatusServiceGetAccountCreationStatusResponse".

(RPC_RESPONSE_STANDARD_NAME)